text.go

```diff
@@ -871,15 +873,15 @@ func (t *Text) Type(r rune) {
 		t.TypeCommit()
 		// go to where ^U would erase, if not already at BOL
 		nnb = 0
-		if t.q0 > 0 && t.file.ReadC(t.q0-1) != '\n' {
+		if t.q0 > 0 && t.file.f.ReadC(t.q0-1) != '\n' {
 			nnb = t.BsWidth(0x15)
 		}
 		t.Show(t.q0-nnb, t.q0-nnb, true)
 		return
 	case 0x05: // ^E: end of line
 		t.TypeCommit()
 		q0 = t.q0
-		for q0 < t.file.Size() && t.file.ReadC(q0) != '\n' {
+		for q0 < t.file.f.Size() && t.file.f.ReadC(q0) != '\n' {
```

Comment:
As I noted elsewhere: you wrote the wrapper functions so this whole line (and many others) wouldn't have needed to change right?